include/fmt/chrono.h

```diff
@@ -290,36 +291,49 @@ inline null<> localtime_s(...) { return null<>(); }
 inline null<> gmtime_r(...) { return null<>(); }
 inline null<> gmtime_s(...) { return null<>(); }
 
+inline const std::locale& get_classic_locale() {
+  static const auto& locale = std::locale::classic();
+  return locale;
+}
+
 template <typename Char>
-inline auto do_write(const std::tm& time, const std::locale& loc, char format,
-                     char modifier) -> std::basic_string<Char> {
-  auto&& os = std::basic_ostringstream<Char>();
+inline void do_write(buffer<Char>& buf, const std::tm& time,
+                     const std::locale& loc, char format, char modifier) {
+  auto&& format_buf = formatbuf<Char>(buf);
+  auto&& os = std::basic_ostream<Char>(&format_buf);
```

Comment:
Why `&&`?

Reply:
Copy from:
https://github.com/fmtlib/fmt/blob/6ab73113fc39e43329b109d57cd079e15465eacd/include/fmt/ostream.h#L100-L101

Reply:
I guess it has something to do with noncopyability.